lmp/tokenizer/_base_dict_tokenizer.py

```diff
@@ -235,9 +249,20 @@ def encode(
                 and pad to `max_seq_len` when sequence length is shorter than
                 `max_seq_len`.
 
+        Raises:
+            TypeError:
+                When `sequence` is not instance of `str`.
```

Comment:
When `sequence` is not instance of `str` or `max_seq_len` is not instance of `int`.